cmd/up/ctx/navigation.go

```
@@ -300,24 +303,33 @@ func (o *Organization) Items(ctx context.Context, upCtx *upbound.Context, navCtx
 			continue
 		}
 
-		ingress, err := navCtx.ingressReader.Get(ctx, space)
-		if err != nil {
-			if errors.Is(err, spaces.SpaceConnectionError) {
-				// we found the space to be unreachable
-				continue
+		wg.Add(1)
+		go func(space upboundv1alpha1.Space) {
+			defer wg.Done()
+			ingress, err := navCtx.ingressReader.Get(ctx, space)
+			if err != nil {
+				if errors.Is(err, spaces.SpaceConnectionError) {
+					// we found the space to be unreachable
+					return
+				}
 			}
```

@RedbackThomson Unchanged in this PR, but is it intentional that we proceed to add an item if we get an error other than `SpaceConnectionError`? I suspect we might NPE below (`*ingress`) in some cases.

---

Hmmm I can't think of a reason why we would want to continue, so this would be an oversight. Ideally we wouldn't want to fail silently, though, since users just wouldn't see the space but also wouldn't know why

---

Made the `continue` unconditional for now, and left a TODO to add unselectable items for the unreachable spaces. I think we want to show them regardless of what kind of error we hit, but may want different wording depending on whether it's a known error (`SpaceConnectionError`) or something else.